ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/dhcp-servers.yml

```diff
@@ -0,0 +1,34 @@
+dhcp_servers:
+  - vrf: TEST
+    ipv4_vendor_options:
+      - vendor_id: NTP
+        sub_options:
+          - code: 42
+            type: ipv4-address
+            data: "10.1.1.1"
+          - code: 1
+            type: string
+            data: "test"
+    disabled: false
+    dns_domain_name_ipv4: testv4.com
+    dns_domain_name_ipv6: testv6.com
+    subnets:
+      - subnet: 10.0.0.0/24 # required
+        name: TEST1
+        ranges:
+          - start: 10.0.0.10 # required
+            end: 10.0.0.100 # required
+          - start: 10.0.0.110 # required
+            end: 10.0.0.120 # required
```

Comment:
lets put ranges out of order to test sorting

Reply:
Fixed.